erts/emulator/ryu/vendor.info

```diff
@@ -14,18 +14,22 @@
     "homepage": "https://github.com/ulfjack/ryu",
     "licenseDeclared": "Apache-2.0",
     "name": "ryu",
-    "versionInfo": "844864ac213bdbf1fb57e6f51c653b3d90af0937",
+    "versionInfo": "1264a946ba66eab320e927bfd2362e0c8580c42f",
```

Comment:
is there any version info? Just to see if we avoid this accidental duplication of sha in `versionInfo` and `sha`.
I think not, but just checking